components/odh-notebook-controller/config/default/kustomization.yaml

```diff
@@ -5,7 +5,6 @@ bases:
   # - ../crd
   - ../rbac
   - ../manager
-  - ../webhook
```

Comment:
Since webhook component is a requirement for odh-notebook-controller, we should reference the webhook manifests as part of notebook controller.

This should also then allow the unit tests to pass.